app/mattermost.android.js

```diff
@@ -0,0 +1,436 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import 'babel-polyfill';
+import {
+    Alert,
+    AppState,
+    InteractionManager,
+    Platform,
+    NativeModules,
+    Keyboard,
+    DeviceEventEmitter,
+} from 'react-native';
+const {StatusBarManager, StartTime} = NativeModules;
+
+import DeviceInfo from 'react-native-device-info';
+import {Navigation} from 'react-native-navigation';
+import {Provider} from 'react-redux';
+import semver from 'semver';
+
+import {Client4} from 'mattermost-redux/client';
+import {General} from 'mattermost-redux/constants';
+import {setAppState, setServerVersion} from 'mattermost-redux/actions/general';
+import {loadMe, logout} from 'mattermost-redux/actions/users';
+import {handleLoginIdChanged} from 'app/actions/views/login';
+import {handleServerUrlChanged} from 'app/actions/views/select_server';
+import EventEmitter from 'mattermost-redux/utils/event_emitter';
+
+import initialState from 'app/initial_state';
+import configureStore from 'app/store';
+import {NavigationTypes} from 'app/constants';
+import mattermostBucket from 'app/mattermost_bucket';
+import mattermostManaged from 'app/mattermost_managed';
+import PushNotifications from 'app/push_notifications';
+import {registerScreens} from 'app/screens';
+import {
+    calculateDeviceDimensions,
+    setDeviceOrientation,
+    setDeviceAsTablet,
+    setStatusBarHeight
+} from 'app/actions/device';
+import {loadConfigAndLicense, startDataCleanup} from 'app/actions/views/root';
+import {setChannelDisplayName} from 'app/actions/views/channel';
+import {deleteFileCache} from 'app/utils/file';
+import LocalConfig from 'assets/config';
+
+import App from './app';
+
+const AUTHENTICATION_TIMEOUT = 5 * 60 * 1000;
+
+export const app = new App();
+export const store = configureStore(initialState);
+registerScreens(store, Provider);
+
+if (Platform.OS === 'android') {
```

Comment:
Is this needed since the file has an .android.js extension?

Reply:
So the ultimate goal is to support one file `mattermost.js` instead of 2 based on os.

I kept the old mattermost.js file (converted to .ios.js) to support iOS in the meantime while we come up with a plan for testing iOS